homeassistant/components/ozw/websocket_api.py

```diff
@@ -45,6 +46,46 @@ def websocket_get_instances(hass, connection, msg):
     )
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "ozw/get_nodes",
+        vol.Optional(OZW_INSTANCE, default=1): vol.Coerce(int),
+    }
+)
+def websocket_get_nodes(hass, connection, msg):
+    """Get a list of nodes for an OZW instance."""
+    manager = hass.data[DOMAIN][MANAGER]
+    nodes = []
+
+    for node in manager.get_instance(msg[OZW_INSTANCE]).collections["node"]:
+        nodes.append(
+            {
+                "node_query_stage": node.node_query_stage,
```

Comment:
I think we should create constants now for the strings that are shared between commands to ensure we use the same terms.